src/CoreTests/Types/HeadingLevelTypeTests.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Axe.Windows.Core.Types.Tests
+{
+    /// <summary>
+    /// Tests for HeadingLevelType class
+    /// </summary>
+    [TestClass()]
+    public class HeadingLevelTypeTests
+    {
+        [TestMethod()]
+        public void HeadingLevelNone()
+        {
+            Assert.AreEqual("HeadingLevel_None(80050)", HeadingLevelType.GetInstance().GetNameById(HeadingLevelType.HeadingLevelNone));
+        }
+
+        [TestMethod()]
+        public void HeadingLevel1()
+        {
+            Assert.AreEqual("HeadingLevel1(80051)", HeadingLevelType.GetInstance().GetNameById(HeadingLevelType.HeadingLevel1));
```

Comment:
@RobGallo, do you think we should use a friendlier name here, say "Heading Level 1" or maybe just "Level 1" ? The values I've chosen here match the documentation values, but aren't necessarily friendly